test/e2e/common/setup-kubestellar.sh

```diff
@@ -129,12 +129,36 @@ popd
 : Waiting for OCM hub to be ready...
 if [ "$use_release" = true ] ; then
     kubectl wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.its-with-clusteradm}=true' --timeout 400s
-    kubectl wait -n its1-system job.batch/its-with-clusteradm --for condition=Complete --timeout 400s
+    kubectl wait -n its1-system job.batch/its-with-clusteradm --for condition=Complete --timeout 400s || {
+        # Handle backoffLimit retry case: check if any pod succeeded
+        if kubectl -n its1-system get pods -l job-name=its-with-clusteradm --field-selector=status.phase=Succeeded -o name | grep -q .; then
+            echo "Job succeeded (some pods completed), continuing..."
+        else
+            echo "Job failed (no pods succeeded)"
+            exit 1
+        fi
+    }
```

Comment:
I do not understand the point of this. The Job only specifies one Pod. The Job will succeed if any try of that Pod succeeds. This search for a successful Pod is redundant.